enforce_codeowners/keras_bot/pull_requests.py

```diff
@@ -29,15 +33,31 @@ def parse_codeowners():
     return map_path_owner
 
 
-def send_message(pull_request, owner, files_changed):
+def send_message(pull_request, owner, files_changed, additions):
     message = """Hello, I'm a bot! I can make mistakes, notify gabrieldemarmiesse if I made one.
 
 I see that you modified the following file{plural}: 
 {files_changed}
 
 The owner of those file is @{owner}
+"""
 
-@{owner} could you please take a look at it whenever 
+    absKeys = list(additions['absolute'].keys())[:10]
+    relKeys = list(additions['relative'].keys())[:10]
+
+    absolute = "The following files should not contain relative imports\n"
+    for file in range(len(absKeys)):
+        absolute += absKeys[file] + "\n"
+    absolute += ("\nThis is " + str(file+1) + " out of " + len(additions['absolute']) + " files with relative imports to be changed\n")

```

Comment:
prefer f-string formatting when possible. The docker image used is with python 3.7 so we're safe.